src/config.ts

```diff
@@ -1,5 +1,5 @@
 // TODO: Eventually abstract this out of this repo
 // My .env trick I've used in other projects needs some touching up of its regex
 export default {
-  SERVER_HOSTNAME: 'https://roguelikecelebration-mud.azurewebsites.net'
+  SERVER_HOSTNAME: 'https://roguelikecelebration.azurewebsites.net'

```

Comment:
This is your personal dev backend, yeah?

Reply:
Yup sorry, meant to not include this one in the CL, I'll get it taken out